connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala

```diff
@@ -459,6 +464,25 @@ class SparkBigQueryEndToEndITSuite extends FunSuite
     initialDataValuesExist shouldBe true
   }
 
+  test ("write to bq - partitioned and clustered table") {
+    val df = spark.read.format("com.google.cloud.spark.bigquery")
+      .option("table", LIBRARIES_PROJECTS_TABLE)
+      .load()
+      .where("platform = 'Sublime'")
+
+    df.write.format("bigquery")
+      .option("table", fullTableNamePartitioned)
+      .option("temporaryGcsBucket", temporaryGcsBucket)
+      .option("partitionField", "created_timestamp")
+      .option("clusteredFields", "platform")
+      .mode(SaveMode.Overwrite)
+      .save()
+
+    val tableDefinition = testPartitionedTableDefinition
+    tableDefinition.getTimePartitioning.getField shouldBe "created_timestamp"
+    tableDefinition.getClustering.getFields should contain ("platform")
+  }
```

Comment:
Please make sure that the partitioned table is deleted when the test ends

Reply:
@davidrabinowitz Thanks for the comments, I fixed all of them, except this one, the table is created inside the test dataset, and this dataset is fully deleted inside the afterAll method, should I explicitly delete the table before deleting the dataset? When I ran the tests, the table didn't exists after the dataset was deleted